documentation/source/BenchmarkingYoloNAS.md

```diff
@@ -0,0 +1,45 @@
+# Benchmarking YoloNAS
+## Introduction:
+
+YoloNAS is a leading object detection architecture that combines accuracy and efficiency. Using quantization-aware training (QAT), YoloNAS models can be optimized for resource-constrained devices.
```

Comment:
I would write:
```
"Using [post training quantization (PTQ)](ptq_qat.md) and [quantization-aware training (QAT)](ptq_qat.md)"
```

Reply:
done in 3a0181b3dd24ba7b8091b1b8e1e01a1fee562ce0